### PR TITLE
Add a Gitter chat badge to README.md

### PR DESCRIPTION
**Pull request by gitter-badger**
### SciTools/iris now has a Chat Room on Gitter

@lbdreyer has just created a chat room. You can visit it here: [https://gitter.im/SciTools/iris](https://gitter.im/SciTools/iris?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&content=body_link).

This pull-request adds this badge to your README.md:

[Gitter](https://gitter.im/SciTools/iris?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge)

If my aim is a little off, please [let me know](https://github.com/gitterHQ/readme-badger/issues).

Happy chatting.

PS: [Click here](https://gitter.im/settings/badger/opt-out) if you would prefer not to receive automatic pull-requests from Gitter in future.

### CODE REVIEW

**Comment by lbdreyer** [comment]
It turns out we already had a gitter, but did I start an Iris specific room on the gitter. This just adds the badge to the readme.

Could someone merge this in? @pp-mo @bjlittle @dkillick @djkirkham 

Once this is merged in I will put a post on google groups notifying people about this

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Iris
 ====
 
+[![Join the chat at https://gitter.im/SciTools/iris](https://badges.gitter.im/SciTools/iris.svg)](https://gitter.im/SciTools/iris?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
```
Comment (dkillick) [inline README.md]: Can we have this inline with the other badges please?

Comment (lbdreyer) [inline README.md]: I guess it's that blank line on line number 5 which is separating the badge onto it's own line. 
I'm not sure how to modify this PR so I will close this one and open another one with my own branch

Comment (dkillick) [inline README.md]: Maybe this is an example of where the badger's aim is a little off?

Comment (dkillick) [inline README.md]: Alternatively we merge this and then add a follow-up PR to remove the blank line...

Comment (lbdreyer) [inline README.md]: Same amount of work. The new PR is now up #2510

**Pull request closed without merging**